ivy_tests/test_ivy/test_frontends/test_jax/test_jax_src_api.py

```diff
@@ -4,6 +4,83 @@
 import ivy_tests.test_ivy.helpers as helpers
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 from ivy.functional.frontends.jax._src.api import device_put, device_get
+from hypothesis import given, assume, strategies as st
+from ivy.functional.frontends.jax._src.api import vmap
+
+
+def _fn1(x, y):
+    return ivy.matmul(x, y)
+
+
+def _fn2(x, y):
+    return ivy.vecdot(x, y)
+
+
+def _fn3(x, y):
+    return ivy.add(x, y)
+
+
+# vmap
+@handle_frontend_test(
+    fn_tree="jax._src.api.vmap",
+    func=st.sampled_from([_fn1, _fn2, _fn3]),
+    dtype_and_arrays_and_axes=helpers.arrays_and_axes(
+        allow_none=False,
+        min_num_dims=2,
+        max_num_dims=5,
+        min_dim_size=2,
+        max_dim_size=10,
+        num=2,
+        return_dtype=True,
+    ),
+    in_axes_as_cont=st.booleans(),
+)
```

Comment:
Also I would suggest sampling a backend string here and passing it to `test_vmap` so you can set a random backend in each iteration of the test.

Reply:
Make sense!